examples/txnkv/keyspace.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2025 TiKV Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pingcap/kvproto/pkg/kvrpcpb"
+	"github.com/tikv/client-go/v2/txnkv"
+)
+
+var (
+	pdAddr       = flag.String("pd", "127.0.0.1:2379", "comma-separated PD address list")
+	keyspaceName = flag.String("keyspace", "", "keyspace name (required for API v2)")
+)
+
+func main() {
```

Comment:
Perhaps add some comments about the required deployment including next-gen tikv-server and pd-server with next-gen build flag